datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/jmx/{check_name}/README.md

```diff
@@ -16,16 +16,16 @@ This check monitors [{integration_name}][1].
    Agent's configuration directory to start collecting your {check_name} performance data.
    See the [sample {check_name}.d/conf.yaml][2] for all available configuration options.
 
-   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the info page.
-   You can specify the metrics you are interested in by editing the configuration below.
-   To learn how to customize the metrics to collect visit the [JMX Checks documentation][3] for more detailed instructions.
-   If you need to monitor more metrics, contact [Datadog support][4].
+   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated when running the Datadog Agent [status command][3].
+   You can specify the metrics you are interested in by editing the [configuration][2].
+   To learn how to customize the metrics to collect visit the [JMX Checks documentation][4] for more detailed instructions.
+   If you need to monitor more metrics, contact [Datadog support][5].
 
 2. [Restart the Agent][5]
```

Comment:
This is now link 6. If you keep the 5 here, it will link to 5.

Reply:
Updated!